Add crio-status client for convenient HTTP API access

saschagrunert opened the pull request:
The client API is now internal but we can provide a simple command line
interface for it.

vrothberg commented:
> Code LGTM but tests and man pages are missing.

And bash-completions would be great.

TomSweeneyRedHat commented on cmd/crio-status/main.go:
```diff
@@ -0,0 +1,160 @@
+		Aliases: []string{"c"},
+		Flags:   flags,
+		Name:    "config",
+		Usage:   "retrieve the configuration as TOML string",
```
nit suggestion  s/as TOML/as a TOML/

saschagrunert replied:
Fixed as suggested.

saschagrunert commented:
> Code LGTM but tests and man pages are missing.

Added it. :+1: 

> And bash-completions would be great.

Can we tackle this later on? The [urfave/cli](https://github.com/urfave/cli#bash-completion) generator offers only a two-step generation where I'm not sure if we want this... Maintaining seems also not nice to me, WDYT?

vrothberg commented:
> Can we tackle this later on? The [urfave/cli](https://github.com/urfave/cli#bash-completion) generator offers only a two-step generation where I'm not sure if we want this... Maintaining seems also not nice to me, WDYT?

Why not generate it now and add it? I think it's better to have a simple completion than none. If you prefer to add this later, please open an issue so we don't forget about it. My squirrel brain would certainly :^)

saschagrunert commented:
> Why not generate it now and add it? I think it's better to have a simple completion than none. If you prefer to add this later, please open an issue so we don't forget about it. My squirrel brain would certainly :^)

Added the bash completion and adapted the install scripts. As a little extra I added a fish completion, too! :heart_eyes:

saschagrunert commented:
/test e2e_rhel

saschagrunert commented:
/test integration_fedora

saschagrunert commented:
/test integration_fedora

saschagrunert commented:
/test integration_fedora

saschagrunert commented:
/test integration_fedora

saschagrunert commented:
/retest

saschagrunert commented:
/retest

rhatdan commented:
/lgtm

saschagrunert commented:
/retest

saschagrunert commented:
/test e2e
/test kata

haircommander commented:
/retest

saschagrunert commented:
/retest

saschagrunert commented:
/retest
